tests/test_wheel.py

```diff
@@ -1,52 +1,224 @@
+import json
 import zipfile
+from copy import deepcopy
+from dataclasses import asdict, dataclass
 from pathlib import Path
-from typing import TYPE_CHECKING
+from tempfile import TemporaryDirectory
 
-import pkginfo
+import build
 import pytest
+from test_spec import LOCK_EXAMPLE
```

Comment:
would make this a relative import

Reply:
or move whatever it's using to `conftest.py` fixtures

Reply:
relative imports don't work in pytest I think?

Reply:
Oh I've put test fixtures into conftest.py